src/components/form/dt-location-map/dt-location-map-item.js

```diff
@@ -162,32 +162,45 @@ export default class DtLocationMapItem extends LitElement {
         }
 
         .field-container {
+          --dt-location-map-border-radius: var(--dt-form-border-radius, 10px);
           display: flex;
           margin-bottom: 0.5rem;
         }
+        .field-container > *:first-child {
+          border-start-start-radius: var(--dt-location-map-border-radius, 0);
+          border-end-start-radius: var(--dt-location-map-border-radius, 0);
+        }
+        .field-container > *:last-child {
+          border-start-end-radius: var(--dt-location-map-border-radius, 0);
+          border-end-end-radius: var(--dt-location-map-border-radius, 0);
+        }
         .field-container input {
           flex-grow: 1;
+          border-width: var(--dt-form-border-width, 1px);
         }
         .field-container .input-addon {
           flex-shrink: 1;
           display: flex;
           justify-content: center;
           align-items: center;
           aspect-ratio: 1/1;
-          padding: 10px;
-          border: solid 1px gray;
+          padding: 0.6em;
           border-collapse: collapse;
           color: var(--dt-location-map-button-color, #cc4b37);
           background-color: var(--dt-location-map-background-color, buttonface);
-          border: 1px solid var(--dt-location-map-border-color, #fefefe);
-          border-radius: var(--dt-location-map-border-radius, 0);
+          border: var(--dt-form-border-width, 1px) solid var(--dt-location-map-border-color, #fefefe);
```

Comment:
I know the color was already defined like this, but I had a though recently that we need to better define the fallback to the global variables. I think it would be good to define `--dt-location-map-border-color` in the root of the element and make sure it has a fallback to `-dt-form-border-color`. Right now it doesn't inherit that global border-color at all.